### PR TITLE
13 identify and use a model fine tuned for classification task

### LINKED ISSUE #13: Identify and use a model fine-tuned for classification task
- [ ] **Model Selection**: Choose an appropriate deep learning model that is well-suited for classification tasks. 

- [ ] **Model Training**: Train the selected model on a prepared dataset. This involves choosing a suitable loss function and optimizer (if using neural networks), and setting appropriate training parameters.

- [ ] **Model Evaluation**: Evaluate the model’s performance using relevant metrics such as accuracy, precision, recall, and F1-score on the validation set to ensure the model reliably classifies issues.

Model suggestions:
- BERT: [google-bert/bert-large-uncased](https://huggingface.co/google-bert/bert-base-uncased) (5.0 GB memory required)
- ELECTRA [google/electra-large-generator](https://huggingface.co/google/electra-large-generator) + [google/electra-large-discriminator](https://huggingface.co/google/electra-large-discriminator) (0.8 + 5.0 GB)

### PR DESCRIPTION
**Pull request by karma-riuk**
Closes #13

### CODE REVIEW

**Review by lorenzovarese — APPROVED** [review]
lgtm
